configure.ac

```diff
@@ -19,6 +20,7 @@ AC_CHECK_HEADERS([arpa/inet.h fcntl.h libintl.h limits.h locale.h malloc.h \
 		  syslog.h termios.h])
 
 dnl Checks for typedefs, structures, and compiler characteristics.
+AC_C_BIGENDIAN
```

Comment:
I do get `configure.ac:783: warning: AC_C_BIGENDIAN should be used with AC_CONFIG_HEADERS` on ./bootstrap.sh.

Reply:
Yes, that will require inclusion of a "config.h" file that I'd discussed with you a while ago. Those changes will be pushed as part of a separate changeset, after this PR.

Reply:
Ah, you're right. I think I switched away from using the autoconf-defined value when refactoring to use `platform.h` everywhere, because that header is platform-aware.

`__BIG_ENDIAN__` is predefined by xlC on AIX, while on Linux it defines `__LITTLE_ENDIAN__`. Since this header is now only used on AIX, I'll get rid of the `autoconf` macro.